server/tests/api/videos/video-transcoder.ts

```diff
@@ -91,6 +93,93 @@ describe('Test video transcoding', function () {
     expect(torrent.files[0].path).match(/\.mp4$/)
   })
 
+  it('Should transcode high bit rate mp3 to proper bit rate', async function () {
+    this.timeout(60000)
+
+    console.log('is streamable?')
+
+    const videoAttributes = {
+      name: 'my super name for server 2',
```

Comment:
You don't need a particular name for this test, you can remove it (same for the description and other tests)